File: Carthage/Checkouts/SwiftTheme/OCDemo/AboutCell.m
//
//  AboutCell.m
//  SwiftTheme
//
//  Created by Gesen on 16/5/26.
//  Copyright © 2016年 Gesen. All rights reserved.
//

#import "AboutCell.h"
#import "Global.h"
#import "OCDemo-Swift.h"

@implementation AboutCell

- (void)awakeFromNib {
    [super awakeFromNib];
    
    self.content.theme_textColor = globalTextColorPicker;
}

@end

File: Carthage/Checkouts/SwiftTheme/PlistDemo/AboutCell.swift
//
//  AboutCell.swift
//  PlistDemo
//
//  Created by Gesen on 16/3/2.
//  Copyright © 2016年 Gesen. All rights reserved.
//

import UIKit

class AboutCell: BaseCell {

    @IBOutlet weak var content: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()

        content.theme_textColor = "Global.textColor"
    }

}
